src/main/java/uk/gov/hmcts/darts/audio/validation/MediaIdValidator.java

```diff
@@ -13,6 +13,12 @@ public class MediaIdValidator implements Validator<Integer> {
 
     private final MediaRepository mediaRepository;
 
+    public void validateNotHidden(Integer id) {
+        if (mediaRepository.findByIdAndIsHiddenFalse(id).isEmpty()) {
```

Comment:
It maybe more efficient to use existsByIdAndIsHiddenFalse(...) just so eager loading does not kick in

Reply:
Done